src/handlers/gasPriceManager.ts

```diff
@@ -430,14 +444,12 @@ export class GasPriceManager {
         const baseFee = latestBlock.baseFeePerGas
         this.saveBaseFeePerGas(baseFee, Date.now())
 
-        this.logger.debug({
-            baseFee
-        }, "Base fee update");
+        return baseFee
     }
 
     public async getBaseFee(): Promise<bigint> {
-        if (this.queueBaseFeePerGas.length === 0) {
-            await this.updateBaseFee()
+        if (this.gasPriceRefreshIntervalInSeconds === 0) {
+            return this.updateBaseFee()
         }
 
         const {
```

Comment:
What if `this.queueBaseFeePerGas[this.queueBaseFeePerGas.length - 1];` is empty? server start or something like that?

Reply:
yep, the value is pre-load on start

https://github.com/pimlicolabs/alto/pull/271/files#diff-9c1f8157eb2db0945c399ad1ce784c3d0c9f3557b2a9b0768c71ed6cc578824dR177